test/distributed/org/apache/cassandra/distributed/shared/RepairResult.java

```diff
@@ -16,13 +16,16 @@
  * limitations under the License.
  */
 
-package org.apache.cassandra.distributed.api;
+package org.apache.cassandra.distributed.shared;
 
-public interface IListen
+public class RepairResult
```

Comment:
this diff is weird... you added this to replace the usage of `Pair` yet git thinks you replace IListen o_O

Reply:
Hm yes; not sure why / how this happened.